src/utils/optionsparser.cc

```diff
@@ -41,14 +41,14 @@ const int kUciLineIndent = 32;
 const int kHelpWidth = 72;
 }  // namespace
 
-OptionsParser::Option::Option(const OptionId& id) : id_(id) {}
+OptionsParser::Option::Option(const OptionId& id) : id_(id), hidden_(false) {}
```

Comment:
nit: in C++11 it's cleaner to initialize such variables right in declaration: `bool hidden_ = false;`.